src/view/new-address.js

```diff
@@ -34,7 +34,7 @@ const styles = StyleSheet.create({
     marginBottom: 40,
   },
   copied: {
-    bottom: 180,
+    bottom: 60,
```

Comment:
V simple.

Reply:
Setting `color={color.notifyDark}` on `CopiedNotification` in line 60 is probably cleaner: https://github.com/lightninglabs/lightning-app/pull/688/files#diff-7ed4897f600feac8ff2579bda31e993aR60